### ISSUE TITLE
Option to show/unfold all events in month view as in calendar <9

### ISSUE THREAD

**Issue by c-holtermann** [owncloud/calendar#377]
### Steps to reproduce
1. Have more than 4 events per day
2. Use month view
### Expected behaviour

I want to see all events on that day as it had been the case in calendar <9
### Actual behaviour

Three events are being shown and the note that there are more events (+4 events for example)
### Server configuration

**ownCloud version:** 9.0

**Calendar version:** 1.0

**Updated from an older ownCloud or fresh install:** Updated from 8.2

**List of activated apps:**

Enabled:
- activity: 2.2.1
- calendar: 1.0
- comments: 0.2
- contacts: 1.1.0.0
- dav: 0.1.5
- documents: 0.12.0
- encryption: 1.2.0
- federatedfilesharing: 0.1.0
- federation: 0.0.4
- files: 1.4.4
- files_external: 0.5.2
- files_pdfviewer: 0.8
- files_sharing: 0.9.1
- files_texteditor: 2.1
- files_trashbin: 0.8.0
- files_versions: 1.2.0
- files_videoplayer: 0.9.8
- firstrunwizard: 1.1
- gallery: 14.5.0
- notifications: 0.2.3
- provisioning_api: 0.4.1
- systemtags: 0.2
- templateeditor: 0.1
- updatenotification: 0.1.0
  Disabled:
- external
- user_external
- user_ldap

**Comment by georgehrke** [comment]
that's a feature, otherwise the month view would beginn to scroll.

**Comment by georgehrke** [comment]
Why is this bad in your opinion?

**Comment by c-holtermann** [comment]
I need the overview. I have a lot of events and the view is about useless for my purpose the way it is now. I have about a dozen calendars for different institutions and I need to be able to see with one glance if there is an important event one day. Scrolling is no problem. I desperately wish to be able to get back the old behaviour. Just now I think I won't update from 8.2 to 9 on my production system to keep the old way.

**Comment by georgehrke** [comment]
And the week view is not suitable?

> Just now I think I won't update from 8.2 to 9 on my production system to keep the old way.

Please keep in mind that the old calendar is completely unmaintained by now.

**Comment by c-holtermann** [comment]
No, week view ist not really suitable. I don't need to know the time of the events. I need to get the overview if there is an event on that day. And I can't see the whole day in week view. I need to scroll. The overview of the day in the first row is the same as in month view. It hides the events from the 4th on. Even if week view would show a complete overview of the week with one glance (which it doens't on my system) it would be impractical because I would need to switch weeks often when reviewing. The task is to find a date with no events in some months range. That would mean reloading the week page multiple times to check every week for a free spot.
Please let me choose to disable this feature ! I very much liked to see all events in month view.

**Comment by c-holtermann** [comment]
And because the old calendar is unmaintained I even more plea for mercy ;-)

**Comment by georgehrke** [comment]
cc @jancborchardt

**Comment by c-holtermann** [comment]
Thank you for listening to my question !

**Comment by jancborchardt** [comment]
How about hovering a day »expands« the current day to show the rest of the events below?

**Comment by georgehrke** [comment]
I think that the '+ n more' is the best solution right now, as we don't want the month view to begin scrolling, but when we have continuous scrolling, we could also show all events, right?

**Comment by jancborchardt** [comment]
Even with continuous scrolling, I would first say that the height should stay consistent. We could expand it  a bit, but when there’s a day with 10 events that should not cause the whole week to go crazy in height.

**Comment by c-holtermann** [comment]
I now understand that it is a design decision to not scroll. And that the height should stay the same.
I don't understand the reasoning behind these decisions though. At least for my simple mind I was better with scrolling and weeks with different heights and I could see everything.
I like the suggestion by jancborchardt with hovering better then the way it is now.
I don't understand "continous scrolling". What does that mean in this context ?
When I try to find a way to live with the situation as it is now: If I want to see if there is an event that collides with my planning I need to click one day after the other to unfold. There I would wish to be able to click on the next day to expand it. But the field to expand the events is being covered and blocked by the expanded events of the day before (at least in my resolution). So I need to close it first and then click on the next days expand which is tedious. It would be easier if there was a small expand symbol in the row with name of the day. Like |      Fr     X | X ist the expand symbol. This area is never being covered by the unfolding events of the cell right or left. So I could just click on one day after the other and check them through. 

( It's a bit like opening doors in a christmas calendar and I'd prefer to have a button "unfold all" though ;-) and I feel a bit like Oskar the grouch when saying this )

**Comment by georgehrke** [comment]
@c-holtermann Is maybe a list-view (https://github.com/owncloud/calendar/issues/85) something that fits your needs?

**Comment by brucewillis88** [comment]
Please make this happen, hiding events is unacceptable.
